srfi-doc-nonfree/srfi/scribblings/srfi-std/srfi-5.html

```diff
@@ -1,30 +1,56 @@
-<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 3.2//EN">
-<html>
+<!DOCTYPE html PUBLIC "-//IETF//DTD HTML 3.2//EN" "http://www.w3.org/TR/HTML32.dtd">
+<html lang="en">
 <head>
+   <meta charset="utf-8" />
+<!--
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ This file was automatically transformed for Racket.                      ┃
+┃                                                                          ┃
+┃    Source: https://github.com/LiberalArtist/srfi-5                       ┃
+┃    Commit: 292fb6deffe582994968d1ca9ec120df30450be2                      ┃
```

Comment:
If the numerous pull requests listed in https://github.com/racket/srfi/pull/15/commits/395f9db23488e324f560845104e0e270d8bc05bd are merged before this one, I'd regenerate these files from the upstream repositories and amend these commits.